lib/matplotlib/backends/_backend_tk.py

```diff
@@ -397,6 +410,13 @@ def __init__(self, canvas, num, window):
             if self.toolbar:
                 backend_tools.add_tools_to_container(self.toolbar)
 
+        # If the window has per-monitor DPI awareness, then setup a poll to
+        # check the DPI change queue, to then update the scaling.
+        dpi_queue = queue.SimpleQueue()
+        if _tkagg.enable_dpi_awareness(int(window.frame(), 16), dpi_queue):
```

Comment:
perhaps call the synonym wm_frame() instead of frame() here, for better googlability?